cypress/integration/counter-with-hooks.spec.js

```diff
@@ -0,0 +1,13 @@
+/// <reference types="cypress" />
+/// <reference types="../../lib" />
+
+import React from 'react'
+import CounterWithHooks from '../../src/counter-with-hooks.jsx'
+
+/* eslint-env mocha */
+describe('CounterWithHooks component', function () {
+  it.skip('works', function () {
```

Comment:
can I ask why this test is skipped - is it not working yet?

Reply:
Related to #51 I wasn't able to get it working. Leaving it there as a stub for now

Reply:
gotcha